lib/pengine/common.c

```diff
@@ -70,22 +70,25 @@ static pcmk__cluster_option_t pe_opts[] = {
     {
         XML_CONFIG_ATTR_SHUTDOWN_LOCK, NULL, "boolean", NULL,
         "false", pcmk__valid_boolean,
-        N_("Whether to lock resources to a cleanly shut down node"),
-        N_("When true, resources active on a node when it is cleanly shut down "
+        "Whether to lock resources to a cleanly shut down node",
+        "When true, resources active on a node when it is cleanly shut down "
```

Comment:
Why drop the internationalization?

Reply:
I copied this from the controller, and it's not internationalized there. Likewise, I now see that I copied the shutdown-lock-limit entry WITH internationalization from the scheduler metadata to the controller metadata.

Will internationalization work in the controller metadata? I'd assume so.

Reply:
> Will internationalization work in the controller metadata? I'd assume so.

Disregard, dumb question, it's already used there for other options